insights/parsers/tests/test_krb5.py

```diff
@@ -0,0 +1,71 @@
+from insights.parsers import krb5
+from insights.tests import context_wrap
+
+
+KRB5CONFIG = """
+# Configuration snippets may be placed in this directory as well
+includedir /etc/krb5.conf.d/
+
+[logging]
+ default = FILE:/var/log/krb5libs.log
+ kdc = FILE:/var/log/krb5kdc.log
+ admin_server = FILE:/var/log/kadmind.log
+
+[realms]
+ dns_lookup_realm = false
+ ticket_lifetime = 24h
+ default_ccache_name = KEYRING:persistent:%{uid}
+ EXAMPLE.COM = {
```

Comment:
I notice you've only got one kdc here.  The krb5.conf file can define multiple, e.g.:
```ini
[realms]
  REDHAT.COM = {
   kdc = kerberos01.core.prod.int.phx2.redhat.com.:88
   kdc = kerberos.rdu.redhat.com.:88
   kdc = kerberos.bos.redhat.com.:88
   kdc = kerberos.brq.redhat.com.:88
   admin_server = kerberos.corp.redhat.com.:749
   default_domain = redhat.com
  }
```
How does the code handle that?